plugins/cilium-cni/cni-install.sh

```diff
@@ -36,6 +38,11 @@ while test $# -gt 0; do
       CNI_EXCLUSIVE=$(echo "$1" | sed -e 's/^[^=]*=//g')
       shift
       ;;
+    --log-file*)
+      # shellcheck disable=SC2001
+      LOG_FILE=$(echo "$1" | sed -e 's/^[^=]*=//g')
```

Comment:
I know you're just copy pasta'ing here, but can we get some comments on what this sed is doing?

Reply:
yup, from my understanding it's for the case we run with --log-file=file.log and also for --log-file file.log.

(Yup, I copied from other two fields 😅 )

Reply:
Should we add this as a comment in the code?